pkg/controller/namespace/deletion/namespaced_resources_deleter.go

```diff
@@ -487,6 +513,16 @@ func estimateGracefulTerminationForPods(kubeClient clientset.Interface, ns strin
 	return estimate, nil
 }
 
+func unstructuredToPod(obj *runtime.Unstructured) (*api.Pod, error) {
```

Comment:
Btw, the test to verify that pods are deleted when namespace is deleted: https://github.com/kubernetes/kubernetes/blob/2319c020b9a7bedcee68bff97eecf28f2c383366/test/e2e/namespace.go#L80 passed on this PR. So this is working fine. Wanted to make sure that it makes sense. Using dynamic client seems fine to me.